packages/nextjs/src/tools/codegen/utils.ts

```diff
@@ -207,31 +209,31 @@ export const resolveComponentImportFiles = (
 export const sendCode = async ({
   file,
   token,
-  edgeUrl,
+  targetUrl,
 }: {
   file: ExtractedFile;
   token: string;
-  edgeUrl?: string;
+  targetUrl: string;
```

Comment:
should we define separate interface for these params?  Also, check for missing token or targetUrl

Reply:
targetUrl will always be passed, it is required. Token also. Both are validated in `extractFiles`